### PR TITLE
add accept parameter to SimpleQueue class

### PR DESCRIPTION
**Pull request by lsaavedr**
please add 'accept' parameter to SimpleQueue class

### CODE REVIEW

**Comment by matusvalo** [comment]
Hi @drsantos20. You can start adding tests here: https://github.com/celery/kombu/blob/60dc3456368a06ed4125ea9eac9c06e7fcf5cffe/t/unit/test_simple.py

I would put it to Both test_SimpleQueue and test_SimpleBuffer classes.

**Comment by matusvalo** [comment]
I have added unittests and fixed slightly the PR. Please review. @celery/core-developers

File: kombu/simple.py [hunk 1]
```diff
@@ -134,13 +135,13 @@ def __init__(self, channel, name, no_ack=None, queue_opts=None,
...
             routing_key = queue.routing_key
-        consumer = messaging.Consumer(channel, queue)
+        consumer = messaging.Consumer(channel, queue, accept=accept)
         producer = messaging.Producer(channel, exchange,
                                       serializer=serializer,
                                       routing_key=routing_key,
                                       compression=compression)
         super().__init__(channel, producer,
-                         consumer, no_ack, **kwargs)
+                         consumer, no_ack, accept)
```
Comment (maybe-sybr) [inline kombu/simple.py:144]: Why has this lost the `**kwargs` passthrough? We still accept them in the method signature so I assume we still need to pass them along to be cooperative.

File: kombu/simple.py [hunk 2]
```diff
@@ -67,7 +68,7 @@ def get(self, block=True, timeout=None):
                 remaining = timeout - elapsed
 
     def get_nowait(self):
-        m = self.queue.get(no_ack=self.no_ack)
+        m = self.queue.get(no_ack=self.no_ack, accept=self.accept)
```
Comment (maybe-sybr) [inline kombu/simple.py:71]: Could we avoid this by simply using the `self.consumer.accept` attribute? Then we wouldn't need the new kwarg for this class.

**Review by maybe-sybr — CHANGES_REQUESTED** [review]
A few minor changes which I think would make this better. I'd also like to see a test for what happens if we pass in an empty list/set for the `accept` kwarg.

File: kombu/simple.py (hunk 2 above)
Comment (matusvalo) [reply kombu/simple.py]: Yes I think it is reasonable.

**Comment by matusvalo** [comment]
> I'd also like to see a test for what happens if we pass in an empty list/set for the accept kwarg.
done.

@maybe-sybr I have implemented your comments.

**Review by maybe-sybr — APPROVED** [review]
lgtm, thanks @matusvalo

**Comment by maybe-sybr** [comment]
We haven't done an RC for kombu like we have with celery, so this could arguably land but I'll leave that to @thedrow since he's managing the releases.
